samples/OcelotGateway/Properties/launchSettings.json

```diff
@@ -1,12 +1,19 @@
 {
+  "iisSettings": {
```

Comment:
I don't think we need to add IIS settings in here

Reply:
@thangchung  your mean remove it